kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -159,10 +159,6 @@ window.ExerciseView = Backbone.View.extend({
 
         // TODO-BLOCKER(jamalex): does this need to wait on something, to avoid race conditions?
         _.defer(this.khan_loaded);
-        if (Khan.loaded) {
```

Comment:
Might prefer to do this as an if/else?

If Khan.loaded exists, we should use that to avoid race conditions (this only happens for Khan Exercises, not Perseus), otherwise, we can just do the _.defer.